### PR TITLE
Fix compilation warning

### PR DESCRIPTION
**Pull request by RomainMuller**
Add missing protocol delcaration.

### CODE REVIEW

File: gitx_askpasswd_main.m [hunk 1]
```diff
@@ -320,7 +319,7 @@ int	main( int argc, const char* argv[] )
 		url=[args objectAtIndex:[args count]-1];
 		prompt=[NSString stringWithFormat:@"%@ %@",[NSString stringWithCString:argv[1] encoding:NSASCIIStringEncoding],url];
 		remember=YES;
-	}else if((sizeof(argv)/sizeof(char*))>1){
```
Comment (RomainMuller) [inline gitx_askpasswd_main.m]: My version seems so trivially better than I'm wondering if I'm missing something here...

**Comment by RomainMuller** [comment]
I merged it up to the "new" gitx_ask_password, which apparently introduced a LOT of whitespace changes that made it hell to merge. Anyway... It still needs to be made buffer_overflow-safe.

**Comment by RomainMuller** [comment]
OK, new ask_password doesn't have the buffer vulnerability, but there's a warning that can be removed easily :)

**Comment by laullon** [comment]
I'm working on a new ask_password

**Pull request closed without merging**